src/Swarrot/Broker/MessagePublisher/PublishConfirmPublisherInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+declare(strict_types=1);
```

Comment:
should be removed. We need to be compatible with PHP 5 (and btw, this is totally useless for an interface, as it is not doing any method call)